packages/client/src/components/Modals/GrantDetails.vue

```diff
@@ -166,18 +166,27 @@ export default {
       agencies: 'agencies/agencies',
       users: 'users/users',
       interestedCodes: 'grants/interestedCodes',
+      user: 'users/loggedInUser',
     }),
     alreadyViewed() {
       if (!this.selectedGrant) {
         return false;
       }
-      return this.selectedGrant.viewed_by_agencies.find((viewed) => viewed.agency_id.toString() === this.selectedAgencyId);
+      return this.selectedGrant.viewed_by_agencies.find(
+        (viewed) => viewed.agency_id.toString() === this.selectedAgencyId,
+      );
+    },
+    shouldShowSpocButton() {
+      if (!this.user.tenant.uses_spoc_process) return false;
```

Comment:
might be useful to create a tenant selector

Reply:
@Rapol can you elaborate? Do you mean having interested tenants as well as agencies?

Reply:
Not sure I understand the full context here, but users shouldn't be tracking interest in grants on the tenants level.

Reply:
@davidbkay Sorry David that was super vague. My fault.

Im thinking that this tenant property in the user will be reused in other places. If so it might be better to create a new selector to retrieve the tenant.

We would define a new selector in the user vuex module called `getCurrentTenant()`. We can then import that as a computed field

```
computed: {
    ...mapGetters({
      currentTenant: 'users/getCurrentTenant',
    }),
```

Usage of it in the component

```
shouldShowSpocButton() {
      if (!this.currentTenant.uses_spoc_process) return false;
}
```

Reply:
@Rapol yeah I'm totally down with that. I'll make associated updates.

Reply:
@mhuang-usdr This change is for hiding the generate spoc pdf button that was request by nevada. This pdf is for filling out the required document to start tenant grant application process in their internal workflow.